blitz_api/tests/tests_model_User.py

```diff
@@ -0,0 +1,29 @@
+from rest_framework.test import APITestCase
+
+from blitz_api.factories import UserFactory
+from blitz_api.models import User
+
+
+class UserTests(APITestCase):
+
+    def test_generate_tomato_field_matrix(self):
+        """
+        test that matrix generation works and cells that need to be filled
+        are.
+        """
+        user = UserFactory()
+        matrix = user.generate_tomato_field_matrix()
+        self.assertEqual(len(matrix), User.TOMATO_MATRIX_COLUMN_SIZE)
+        self.assertEqual(len(matrix[0]), User.TOMATO_MATRIX_LINE_SIZE)
+
+        choices = [c['tile'] for c in User.TOMATO_MATRIX_CELLS]
+        for m in range(0, len(matrix)):
```

Comment:
`m` and `n` are valid variable, but it's always better with `column` and `line` for readers ;)